lingetic-nextjs-frontend/app/components/questions/FillInTheBlanks/ActionButton.tsx

```
@@ -0,0 +1,43 @@
+}: ActionButtonProps) {
+  return (
+    <div className="flex justify-end items-center gap-3">
+      <Speaker question={question} autoplay />
+      {isChecked ? (
+        NextButton
+      ) : (
+        <button
+          disabled={isChecking}
+          onClick={onCheck}
+          className="bg-[#2563eb] text-white px-6 py-3 rounded-lg font-medium hover:bg-blue-700 transition-colors flex items-center"
+        >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)